tests/test_ndarrays_regression.py

```diff
@@ -13,14 +12,7 @@ def no_regen(ndarrays_regression, request):
         pytest.fail("--force-regen should not be used on this test.")
 
 
-def test_usage_workflow(testdir, monkeypatch):
-    """
-    :type testdir: _pytest.pytester.TmpTestdir
-
-    :type monkeypatch: _pytest.monkeypatch.monkeypatch
-    """
-
-    import sys
+def test_usage_workflow(pytester, monkeypatch):
```

Comment:
Doubt: on tests we should not use types? or it will be done later one?

Reply:
We have yet to discuss this, but I'm OK with:

1. Not enforcing type-annotate all fixtures: it is not very useful because the tests are executed anyway.
2. Tell mypy to check even untyped functions, so it at least type-checks the functions internally (this is done with the `check_untyped_defs` setting which is `True`).

@tadeu what do you think?